src/main/java/org/example/homework3/Task1.java

```diff
@@ -3,16 +3,17 @@
 public class Task1 {
 
     public static void main(String[] args) {
-        int a = 120;
-        int b = 60;
+        int a = 10;
+        int b = 6;
+        double c = (double) a/b;
 
         System.out.println(a + b);
 
         System.out.println(a - b);
 
         System.out.println(a * b);
 
-        System.out.println(a / b);
+        System.out.println(c);
```

Comment:
Would be better to add some info to the output, except just a pure number itdelf